docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
```

Comment:
I guess we'll have to decide whether it's best to deploy the docker source to the server from github and build locally, or only deploy the docker-compose and configs, referencing pre-built images from dockerhub.